service/src/main/java/bio/terra/catalog/service/DatasetService.java

```diff
@@ -181,6 +187,39 @@ private DatasetPreviewTable generateDatarepoTable(
                 .getResult());
   }
 
+  private DatasetPreviewTable generateRawlsTable(
+      AuthenticatedUserRequest user, Dataset dataset, String tableName) {
+    Map<String, EntityTypeMetadata> ent =
+        rawlsService.entityMetadata(user, dataset.storageSourceId());
+    EntityTypeMetadata obj = ent.get(tableName);
```

Comment:
I think we should have better names here than `ent` and `obj`. In general we try to use complete words, not abbreviations, for names. Maybe `entityMetadata` for `ent` and `tableMetadata` for `obj`?